src/fusor/fusor.py

```diff
@@ -562,10 +567,16 @@ def _normalized_gene(
         gene_norm_resp = self.gene_normalizer.normalize(query)
         if gene_norm_resp.match_type:
             gene = gene_norm_resp.gene
-            gene_id = gene_norm_resp.normalized_id
             if use_minimal_gene:
-                return Gene(id=gene_id, label=gene.label), None
-            gene.id = gene_id
+                return MappableConcept(
+                    primaryCoding=Coding(
+                        id=gene.primaryCoding.id,
+                        code=gene.primaryCoding.code,
+                        system="https://www.genenames.org/data/gene-symbol-report/#!/hgnc_id/",
```

Comment:
I would recommend not hard coding this

```suggestion
                        system=gene.primaryCoding.system,
```